CMakeLists.txt

```diff
@@ -14,15 +14,26 @@ set(COMBINED_PROJECT TRUE)
 # Library type
 option(STATIC "Set to ON to build xlnt as a static library instead of a shared library" OFF)
 
-# c++ language standard to use
-set(XLNT_VALID_LANGS 11 14 17)
-set(XLNT_CXX_LANG "14" CACHE STRING "c++ language features to compile with")
+# C++ language standard to use
+set(XLNT_CXX_VALID_LANGS 11 14 17)
+set(XLNT_CXX_LANG "14" CACHE STRING "C++ language features to compile with")
 # enumerate allowed values for cmake gui
-set_property(CACHE XLNT_CXX_LANG PROPERTY STRINGS ${XLNT_VALID_LANGS})
-# validate value is in XLNT_VALID_LANGS
-list(FIND XLNT_VALID_LANGS ${XLNT_CXX_LANG} index)
+set_property(CACHE XLNT_CXX_LANG PROPERTY STRINGS ${XLNT_CXX_VALID_LANGS})
+# validate value is in XLNT_CXX_VALID_LANGS
+list(FIND XLNT_CXX_VALID_LANGS ${XLNT_CXX_LANG} index)
 if(index EQUAL -1)
-    message(FATAL_ERROR "XLNT_CXX_LANG must be one of ${XLNT_VALID_LANGS}")
+    message(FATAL_ERROR "XLNT_CXX_LANG must be one of ${XLNT_CXX_VALID_LANGS}")
+endif()
+
+# C language standard to use
+set(XLNT_C_VALID_LANGS 11 17 23)
```

Comment:
Shouldn't we support C99, as this is the default version for C++11?

Reply:
You are right - I fixed this in my latest commits.